packages/react-flicking/src/react-flicking/CrossFlicking.tsx

```diff
@@ -0,0 +1,135 @@
+/*
+ * Copyright (c) 2015 NAVER Corp.
+ * egjs projects are licensed under the MIT license
+ */
+import * as React from "react";
+import ListDiffer from "@egjs/list-differ";
+import {
+  CrossFlicking as VanillaCrossFlicking,
+  NormalRenderingStrategy,
+  CrossFlickingEvent,
+  HoldStartEvent,
+  HoldEndEvent,
+  MoveStartEvent,
+  MoveEvent,
+  MoveEndEvent,
+  WillChangeEvent,
+  ChangedEvent,
+  EVENTS,
+  SIDE_EVENTS,
+  CrossFlickingOptions,
+  FlickingOptions,
+} from "@egjs/flicking";
+
+import { DEFAULT_PROPS } from "./consts";
+import { FlickingProps } from "./types";
+import ReactRenderer, { ReactRendererOptions } from "./ReactRenderer";
+import ReactElementProvider from "./ReactElementProvider";
+import Flicking from "./Flicking";
+
+export interface CrossFlickingProps extends FlickingProps {
+  onSideHoldStart: (e: CrossFlickingEvent<HoldStartEvent<Flicking>>) => any;
+  onSideHoldEnd: (e: CrossFlickingEvent<HoldEndEvent<Flicking>>) => any;
+  onSideMoveStart: (e: CrossFlickingEvent<MoveStartEvent<Flicking>>) => any;
+  onSideMove: (e: CrossFlickingEvent<MoveEvent<Flicking>>) => any;
+  onSideMoveEnd: (e: CrossFlickingEvent<MoveEndEvent<Flicking>>) => any;
+  onSideWillChange: (e: CrossFlickingEvent<WillChangeEvent<Flicking>>) => any;
+  onSideChanged: (e: CrossFlickingEvent<ChangedEvent<Flicking>>) => any;
+}
+
+export const CROSSFLICKING_DEFAULT_PROPS: CrossFlickingProps = {
+  ...DEFAULT_PROPS,
+  onSideHoldStart: (e: CrossFlickingEvent<HoldStartEvent>) => {},
+  onSideHoldEnd: (e: CrossFlickingEvent<HoldEndEvent>) => {},
+  onSideMoveStart: (e: CrossFlickingEvent<MoveStartEvent>) => {},
+  onSideMove: (e: CrossFlickingEvent<MoveEvent>) => {},
+  onSideMoveEnd: (e: CrossFlickingEvent<MoveEndEvent>) => {},
+  onSideWillChange: (e: CrossFlickingEvent<WillChangeEvent>) => {},
+  onSideChanged: (e: CrossFlickingEvent<ChangedEvent>) => {},
+};
+
+class CrossFlicking extends Flicking {
```

Comment:
Is the method available?

Reply:
CrossFlicking extends Flicking in react, and Flicking includes a `[@withFlickingMethods](https://github.com/naver/egjs-flicking/blob/master/src/cfc/withFlickingMethods.ts#L20)` decorator that allows to use existing methods.